docs/source/conf.py

```diff
@@ -1285,7 +1285,8 @@
     # torch.nn.utils.init
     "skip_init",
     # torch.nn.utils.memory_format
-    "convert_conv2d_weight_memory_format",
+    "convert_con2d_weight_memory_format",
+    "convert_conv3d_weight_memory_format",
```

Comment:
Adding new entried in this list is not ok. Please make sure that any new public function is properly documented!

Reply:
I've corrected this mistake "convert_con2d_weight_memory_format". But otherwise I've exactly replicated everything related to the function "convert_conv2d_weight_memory_format". Isn't that enough. Could you point out where I should add documentation if not ?

Reply:
Adding it to the proper rst file should do the trick.
Note that a lot of these legacy functions are not documented properly so it is expected that you should not follow exactly what is done there.

Reply:
Thank you. I did.